### DISCUSSION TITLE
Please add opensource vacation management (Urlaubsverwaltung.cloud)

### DISCUSSION THREAD

**Discussion by snickerweb** [community-scripts/ProxmoxVE#689] [Ideas]
"Urlaubsverwaltung" is a open source vacation management for slef hosting. It comes manually or with a Docker Image.

Homepage: https://urlaubsverwaltung.cloud/
Git: https://github.com/urlaubsverwaltung/urlaubsverwaltung

**Comment by havardthom** [comment]
Use https://github.com/community-scripts/ProxmoxVE/discussions/new?category=request-script
